win32ss/gdi/gdi32/objects/text.c

```diff
@@ -1,3 +1,10 @@
+/*
+ *  COPYRIGHT:        See COPYING in the top level directory
```

Comment:
We're trying to get rid of these unspecific headers. In the future please use this style header with a specific license:
https://github.com/reactos/reactos/blob/master/base/applications/control/control.c#L1-L6